Resources/ArmTemplates/template.json

```diff
@@ -0,0 +1,150 @@
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "appServiceName": {
+                        "value": "[variables('WebAppName')]"
+                    },
+                    "appServicePlanName": {
+                        "value": "[variables('AppServicePlanName')]"
+                    },
+                    "appServicePlanResourceGroup": {
+                        "value": "[variables('AppServicePlanResourceGroup')]"
+                    },
+                    "appServiceType": {
+                        "value": "app"
+                    },
+                    "deployStagingSlot": {
+                        "value": false
+                    },
+                    "appServiceAppSettings": {
+                        "value": [
+                            {
+                                "name": "MSDEPLOY_RENAME_LOCKED_FILES",
+                                "value": "1"
+                            },
+                            {
+                                "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                                "value": "[reference(variables('AppServiceAppInsightsName')).outputs.InstrumentationKey.value]"
+                            },
+                            {
+                                "name": "WEBSITE_RUN_FROM_PACKAGE",
+                                "value": "0"
+                            },
+                            {
+                                "name": "Neo4j__Endpoint__Uri",
+                                "value": "[parameters('Neo4jUrl')]"
+                            },
+                            {
+                                "name": "Neo4j__Endpoint__Username",
+                                "value": "[parameters('Neo4jUsername')]"
+                            },
+                            {
+                                "name": "Neo4j__Endpoint__Password",
+                                "value": "[parameters('Neo4jPassword')]"
+                            }
+                        ]
+                    }
+                }
+            },
+            "dependsOn": [
+                "[variables('AppServiceAppInsightsName')]"
+            ]
+        },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "database-dfc-servicetaxonomy-editor",
+            "type": "Microsoft.Resources/deployments",
+            "resourceGroup": "[parameters('SqlResourceGroupName')]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('BuildingBlocksDfcBaseUrl'), 'SqlServer/sql-database.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "databaseName": {
+                        "value": "dfc-servicetaxonomy-editor"
```

Comment:
I think the name should be environment specific, ie be based on the ResourcePrefix variable